src/components/sideMenu/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { Drawer, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+function SideMenu() {
```

Comment:
This side menu should be the part of layout as it will be on many pages .. ??
Correct me If I am wrong ..